app/knowledge_panels.py

```diff
@@ -39,3 +44,43 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(facet: str, value: str):
+    if facet == "packaging":
+        plural_facet = facet
+        # That how the api read packaging
+    else:
+        plural_facet = p.plural(facet)
+    """
+    Changing other facet to plural because that how the search api works
+    """
+    query = {}
+    if value is not None:
+        query[f"{plural_facet}_tags_en"] = value
+        query["fields"] = "code,product_name,last_editor"
```

Comment:
last_modified_t is interesting also so that we can add the information to the user. (imagine you want to check there isn't any update this week).

Reply:
Is this a last_modified_tag?

Reply:
I think it's a `last_modified_t[ime]` or `last_modified_t[imestamp]`

Reply:
oh! its `last_edit_dates_tags`